docs/guides/admin/docs/upgrade.md

```diff
@@ -6,10 +6,62 @@ In case you need information about how to upgrade older versions of Opencast,
 please refer to [older release notes](https://docs.opencast.org).
 
 1. Stop your current Opencast instance
-2. TBA
+2. Replace Opencast with the new version
+3. Review the configuration changes and adjust your configuration accordingly
+4. [Migrate (or rebuild) Elasticsearch index](#migrate-elasticsearch)
```

Comment:
Only the rebuild needs to happen during runtime, however, Option 1 clearly states this.
I put this before the restart since people should read the whole thing and decide what they want to do before starting Opencast. So I'd rather not link to the options directly. However, I'm gonna make the instructions a little more clear.